### ISSUE TITLE
unable to install pip install -U hrequests[all]

### ISSUE THREAD

**Issue by rakaarfi** [daijro/hrequests#60]
### Python Version
Python 3.12.6

### OS
Windows 11

### Description
I'm using windows 11. I can't install `pip install -U hrequests[all]`, I've also tried `pip install -U hrequests['all']` and the results are the same, this is the result:

### Output:
```
Requirement already satisfied: hrequests[all] in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (0.8.2)
Requirement already satisfied: aioprocessing in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (2.0.1)
Requirement already satisfied: async-class in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (0.5.0)
Requirement already satisfied: click in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (8.1.7)
Requirement already satisfied: faust-cchardet in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (2.1.19)
Requirement already satisfied: gevent in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (24.10.3)
Requirement already satisfied: geventhttpclient in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (2.3.1)
Requirement already satisfied: httpx in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (0.27.2)
Requirement already satisfied: numpy in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (2.1.1)
Requirement already satisfied: orjson in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (3.10.10)
Requirement already satisfied: parse in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (1.20.2)
Collecting playwright==1.34.0 (from hrequests[all])
  Using cached playwright-1.34.0-py3-none-win_amd64.whl.metadata (3.6 kB)
Collecting playwright-stealth (from hrequests[all])
  Using cached playwright_stealth-1.0.6-py3-none-any.whl.metadata (2.2 kB)
Collecting pycryptodome (from hrequests[all])
  Using cached pycryptodome-3.21.0-cp36-abi3-win_amd64.whl.metadata (3.4 kB)
Requirement already satisfied: rich in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (13.9.3)
Requirement already satisfied: selectolax in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (0.3.25)
Requirement already satisfied: urllib3 in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from hrequests[all]) (2.2.3)
Collecting greenlet==2.0.2 (from playwright==1.34.0->hrequests[all])
  Using cached greenlet-2.0.2.tar.gz (164 kB)
  Preparing metadata (setup.py) ... done
Collecting pyee==9.0.4 (from playwright==1.34.0->hrequests[all])
  Using cached pyee-9.0.4-py2.py3-none-any.whl.metadata (1.7 kB)
Requirement already satisfied: typing-extensions in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from pyee==9.0.4->playwright==1.34.0->hrequests[all]) (4.12.2)
Requirement already satisfied: colorama in c:\users\raka arfinarendra\appdata\roaming\python\python312\site-packages (from click->hrequests[all]) (0.4.6)
Requirement already satisfied: zope.event in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from gevent->hrequests[all]) (5.0)
Requirement already satisfied: zope.interface in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from gevent->hrequests[all]) (7.1.1)
INFO: pip is looking at multiple versions of gevent to determine which version is compatible with other requirements. This could take a while.
Collecting gevent (from hrequests[all])
  Using cached gevent-24.10.2-cp312-cp312-win_amd64.whl.metadata (13 kB)
  Using cached gevent-24.10.1-cp312-cp312-win_amd64.whl.metadata (13 kB)
  Using cached gevent-24.2.1-cp312-cp312-win_amd64.whl.metadata (13 kB)
  Using cached gevent-23.9.1-cp312-cp312-win_amd64.whl.metadata (13 kB)
  Using cached gevent-23.9.0.post1-cp312-cp312-win_amd64.whl.metadata (14 kB)
  Using cached gevent-23.7.0.tar.gz (5.8 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
  Using cached gevent-22.10.2-cp312-cp312-win_amd64.whl
Requirement already satisfied: setuptools in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from gevent->hrequests[all]) (75.2.0)
Requirement already satisfied: cffi>=1.12.2 in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from gevent->hrequests[all]) (1.17.1)
Requirement already satisfied: certifi in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from geventhttpclient->hrequests[all]) (2024.8.30)
Requirement already satisfied: brotli in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from geventhttpclient->hrequests[all]) (1.1.0)
Requirement already satisfied: anyio in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from httpx->hrequests[all]) (4.6.2.post1)
Requirement already satisfied: httpcore==1.* in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from httpx->hrequests[all]) (1.0.6)
Requirement already satisfied: idna in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from httpx->hrequests[all]) (3.10)
Requirement already satisfied: sniffio in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from httpx->hrequests[all]) (1.3.1)
Requirement already satisfied: h11<0.15,>=0.13 in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from httpcore==1.*->httpx->hrequests[all]) (0.14.0)
Requirement already satisfied: markdown-it-py>=2.2.0 in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from rich->hrequests[all]) (3.0.0)
Requirement already satisfied: pygments<3.0.0,>=2.13.0 in c:\users\raka arfinarendra\appdata\roaming\python\python312\site-packages (from rich->hrequests[all]) (2.18.0)
Requirement already satisfied: pycparser in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from cffi>=1.12.2->gevent->hrequests[all]) (2.22)
Requirement already satisfied: mdurl~=0.1 in c:\users\raka arfinarendra\appdata\local\programs\python\python312\lib\site-packages (from markdown-it-py>=2.2.0->rich->hrequests[all]) (0.1.2)
Using cached playwright-1.34.0-py3-none-win_amd64.whl (29.1 MB)
Using cached pyee-9.0.4-py2.py3-none-any.whl (14 kB)
Using cached playwright_stealth-1.0.6-py3-none-any.whl (28 kB)
Using cached pycryptodome-3.21.0-cp36-abi3-win_amd64.whl (1.8 MB)
Building wheels for collected packages: greenlet
  Building wheel for greenlet (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [115 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\__init__.py -> build\lib.win-amd64-cpython-312\greenlet
      creating build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\__init__.py -> build\lib.win-amd64-cpython-312\greenlet\platform
      creating build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\leakcheck.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_contextvars.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_cpp.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_extension_interface.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_gc.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_generator.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_generator_nested.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_greenlet.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_greenlet_trash.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_leaks.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_stack_saved.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_throw.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_tracing.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_version.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\test_weakref.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\__init__.py -> build\lib.win-amd64-cpython-312\greenlet\tests
      running egg_info
      writing src\greenlet.egg-info\PKG-INFO
      writing dependency_links to src\greenlet.egg-info\dependency_links.txt
      writing requirements to src\greenlet.egg-info\requires.txt
      writing top-level names to src\greenlet.egg-info\top_level.txt
      reading manifest file 'src\greenlet.egg-info\SOURCES.txt'
      reading manifest template 'MANIFEST.in'
      warning: no previously-included files found matching 'benchmarks\*.json'
      no previously-included directories found matching 'docs\_build'
      warning: no files found matching '*.py' under directory 'appveyor'
      warning: no previously-included files matching '*.pyc' found anywhere in distribution
      warning: no previously-included files matching '*.pyd' found anywhere in distribution
      warning: no previously-included files matching '*.so' found anywhere in distribution
      warning: no previously-included files matching '.coverage' found anywhere in distribution
      adding license file 'LICENSE'
      adding license file 'LICENSE.PSF'
      adding license file 'AUTHORS'
      writing manifest file 'src\greenlet.egg-info\SOURCES.txt'
      copying src\greenlet\greenlet.cpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet.h -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_allocator.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_compiler_compat.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_cpython_compat.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_exceptions.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_greenlet.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_internal.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_refs.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_slp_switch.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_thread_state.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_thread_state_dict_cleanup.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\greenlet_thread_support.hpp -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\slp_platformselect.h -> build\lib.win-amd64-cpython-312\greenlet
      copying src\greenlet\platform\setup_switch_x64_masm.cmd -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_aarch64_gcc.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_alpha_unix.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_amd64_unix.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_arm32_gcc.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_arm32_ios.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_arm64_masm.asm -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_arm64_masm.obj -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_arm64_msvc.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_csky_gcc.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_m68k_gcc.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_mips_unix.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_ppc64_aix.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_ppc64_linux.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_ppc_aix.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_ppc_linux.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_ppc_macosx.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_ppc_unix.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_riscv_unix.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_s390_unix.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_sparc_sun_gcc.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_x32_unix.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_x64_masm.asm -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_x64_masm.obj -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_x64_msvc.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_x86_msvc.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\platform\switch_x86_unix.h -> build\lib.win-amd64-cpython-312\greenlet\platform
      copying src\greenlet\tests\_test_extension.c -> build\lib.win-amd64-cpython-312\greenlet\tests
      copying src\greenlet\tests\_test_extension_cpp.cpp -> build\lib.win-amd64-cpython-312\greenlet\tests
      running build_ext
      building 'greenlet._greenlet' extension
      creating build\temp.win-amd64-cpython-312\Release\src\greenlet
      "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.41.34120\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -DWIN32=1 "-IC:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include" "-IC:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\Include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.41.34120\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" /EHsc /Tpsrc/greenlet/greenlet.cpp /Fobuild\temp.win-amd64-cpython-312\Release\src/greenlet/greenlet.obj /EHsr /GT
      greenlet.cpp
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(831): error C2039: 'use_tracing': is not a member of '_PyCFrame'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(67): note: see declaration of '_PyCFrame'
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(834): error C2039: 'recursion_limit': is not a member of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(115): note: see declaration of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(834): error C2039: 'recursion_remaining': is not a member of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(115): note: see declaration of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(848): error C2039: 'trash_delete_nesting': is not a member of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(115): note: see declaration of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(867): error C2039: 'use_tracing': is not a member of '_PyCFrame'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(67): note: see declaration of '_PyCFrame'
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(870): error C2039: 'recursion_remaining': is not a member of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(115): note: see declaration of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(870): error C2039: 'recursion_limit': is not a member of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(115): note: see declaration of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(881): error C2039: 'trash_delete_nesting': is not a member of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(115): note: see declaration of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(891): error C2039: 'use_tracing': is not a member of '_PyCFrame'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(67): note: see declaration of '_PyCFrame'
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(899): error C2039: 'recursion_limit': is not a member of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(115): note: see declaration of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Temp\pip-install-sde2qt6h\greenlet_d5b700baf6da4b7996d34f6ea5d6ca4f\src\greenlet\greenlet_greenlet.hpp(899): error C2039: 'recursion_remaining': is not a member of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(115): note: see declaration of '_ts'
      src/greenlet/greenlet.cpp(3095): error C2039: 'trash_delete_nesting': is not a member of '_ts'
      C:\Users\Raka Arfinarendra\AppData\Local\Programs\Python\Python312\include\cpython/pystate.h(115): note: see declaration of '_ts'
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.41.34120\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for greenlet
  Running setup.py clean for greenlet
Failed to build greenlet
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (greenlet)
```

**Comment by danielrzad** [comment]
same here

**Comment by daijro** [comment]
Related to #39.

v0.9.0 is under development [here](https://github.com/daijro/hrequests/tree/v0.9.0) and will be published to Pypi within the next few days. It should fully support Python 3.12+.

**Comment by rafaeltgt** [comment]
@rakaarfi @danielrzad I managed to install using python 3.10

**Comment by Nongzhsh** [comment]
> Related to #39.
> 
> v0.9.0 is under development [here](https://github.com/daijro/hrequests/tree/v0.9.0) and will be published to Pypi within the next few days. It should fully support Python 3.12+.

There is also a similar error in v0.9.0, but it seems to be a problem with third-party dependent packages.

**Version:**
Python 3.10.12
WSL2 (Ubuntu 22.04.4 LTS)

`uv pip install hrequests[all]==0.9.0b0`

**Error output:**

```
× Failed to download and build `event==0.4.3`

  ╰─▶ Build backend failed to determine requirements with `build_wheel()` (exit status: 1)

      [stderr]
      Traceback (most recent call last):
        File "<string>", line 14, in <module>
        File "/home/tiger/.cache/uv/builds-v0/.tmpVhluX9/lib/python3.10/site-packages/setuptools/build_meta.py", line 334, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=[])
        File "/home/tiger/.cache/uv/builds-v0/.tmpVhluX9/lib/python3.10/site-packages/setuptools/build_meta.py", line 304, in _get_build_requires
          self.run_setup()
        File "/home/tiger/.cache/uv/builds-v0/.tmpVhluX9/lib/python3.10/site-packages/setuptools/build_meta.py", line 522, in run_setup
          super().run_setup(setup_script=setup_script)
        File "/home/tiger/.cache/uv/builds-v0/.tmpVhluX9/lib/python3.10/site-packages/setuptools/build_meta.py", line 320, in run_setup
          exec(code, locals())
        File "<string>", line 20
          print 'found libevent build directory', best
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?
```

**Comment by daijro** [comment]
Fixed in v0.9.0 :+1:
